api/test/enum.test.js

```diff
@@ -138,6 +138,21 @@ describe('Enum values', () => {
     foundationHeaderPerimeterFeet: { testValue: 113.99475430400001 },
     foundationHeaderHeightMetres: { testValue: 0.23 },
     foundationHeaderHeightFeet: { testValue: 0.7545932000000001 },
+    foundationFloorFloorTypeEnglish: { testValue: 'Slab' },
+    foundationFloorFloorTypeFrench: { testValue: 'Dalle' },
+    foundationFloorAreaMetres: { testValue: 72.6432 },
+    foundationFloorAreaFeet: { testValue: 781.9249472263218 },
+    foundationFloorPerimeterMetres: { testValue: 34.7466 },
+    foundationFloorPerimeterFeet: { testValue: 113.998035144 },
+    // TODO: Uncomment and add appropriate values when issue #315 is resloved
+    // foundationFloorInsulationNominalRsi: { testValue: null },
+    // foundationFloorInsulationNominalR: { testValue: null },
+    // foundationFloorInsulationEffectiveRsi: { testValue: null },
+    // foundationFloorInsulationEffectiveR: { testValue: null },
+    // foundationFloorWidthMetres: { testValue: null },
+    // foundationFloorWidthFeet: { testValue: null },
+    // foundationFloorLengthMetres: { testValue: null },
+    // foundationFloorLengthFeet: { testValue: null },
```

Comment:
Oh no! 🤒 

Is this going to be resolved soon (in the next day or two), or are we expecting this to be here for a while?

Reply:
Yeah, there are null values in the test data. When we get the rest of the data I suspect we will find actual values. The goal of these tests is to show that the Mongo query object returned by these functions is capable of returning a value from the database (it's structure lines up with the structure of the document itself). All the `foundationFloor*` functions use the same query object, so if one of the tests pass we have proof that things are OK, but for completeness it'd be good to get these fixed once the data is available... whenever that is.

Reply:
Okay, so we can't currently search on these values because we can't match on nulls.